### DISCUSSION TITLE
include default field values in initialValues

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#14224] [Feature Requests]
The 'values' provided by v-form do not include default values.

This might be intended (?) but it makes it difficult to work with custom interfaces/hooks that rely on a complete data set being passed through. Although it'd be possible to fetch default values from the db when working with custom extensions, it'd be safer to have them included in the emitted object right away.

Would it be possible to include default field values in initialValues or have v-form emit them just like user inputs?

**Comment by rijkvanzanten** [comment]
> Would it be possible to include default field values in initialValues

This makes the most sense to me 👍🏻 Wanna open a PR for that?

**Comment by ghost** [comment]
does that make sense to you, @rijkvanzanten ? Seems to work just fine :)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
